core/jobs/transforms/validation/config_validation.py

```diff
@@ -24,18 +24,29 @@
 from core.jobs.transforms.validation import base_validation
 from core.platform import models
 
+from typing import Type
+
+MYPY = False
+if MYPY:  # pragma: no cover
+    from mypy_imports import config_models
+
 (config_models,) = models.Registry.import_models([models.NAMES.config])
 
 
 @validation_decorators.AuditsExisting(
     config_models.ConfigPropertySnapshotMetadataModel)
 class ValidateConfigPropertySnapshotMetadataModel(
-        base_validation.BaseValidateCommitCmdsSchema):
+    base_validation.BaseValidateCommitCmdsSchema[
+        config_models.ConfigPropertySnapshotMetadataModel
+    ]
+):
     """Overrides _get_change_domain_class for
     ConfigPropertySnapshotMetadataModel.
     """
 
-    def _get_change_domain_class(self, input_model): # pylint: disable=unused-argument
+    def _get_change_domain_class(
+        self, input_model: config_models.ConfigPropertySnapshotMetadataModel  # pylint: disable=unused-argument
```

Comment:
same as above

Reply:
`_` approach is working fine for test files but it is throwing an lint error for main files ( non-test files ).

Reply:
Let's keep it only for tests then